packages/web-components/src/components/masthead/top-nav.ts

```diff
@@ -575,7 +592,9 @@ class C4DTopNav extends StableSelectorMixin(HostListenerMixin(CDSHeaderNav)) {
                           @slotchange=${handleSlotChange}
                           @keydown="${handleOnKeyDown}"></slot>
                       </div>
-                      <div class="${prefix}--sub-content-left"></div>
+                      <div
+                        part="sub-content-left"
+                        class="${prefix}--sub-content-left"></div>
```

Comment:
Missed a `<div>` element in this file on line 601: `<div class="${caretLeftContainerClasses}">`